R/bs-remove.R

```diff
@@ -0,0 +1,65 @@
+#' Remove or retrieve Sass code from a theme
+#'
+#' @inheritParams bs_theme_update
+#' @param ids a character vector of ids
+#' @export
+#' @examples
+#'
+#' # Remove CSS rules for print and carousels
+#' bs_theme(version = 4) %>%
+#'   bs_remove(c("_print", "_carousel"))
+#'
+#' # Remove BS3 compatibility layer
+#' bs_theme(version = 4) %>%
+#'   bs_remove("bs3compat")
+#'
+#' bs_theme(version = 4) %>%
+#'   bs_retrieve("_print") %>%
+#'   file.edit()
+#'
+bs_remove <- function(theme, ids = character(0)) {
+  ids <- retain_valid_ids(theme, ids)
+  sass_bundle_remove(theme, ids)
+}
+
+#' @rdname bs_remove
+#' @param include_unnamed whether or not to include unnamed [sass::sass_layer()]s
+#' (e.g., Bootstrap Sass variables, functions, and mixins).
+#' @export
+bs_retrieve <- function(theme, ids = character(0), include_unnamed = TRUE) {
```

Comment:
My take would be to set `include_unnamed = FALSE` as I would be looking for a particular set of ids

```suggestion
bs_retrieve <- function(theme, ids = character(0), include_unnamed = FALSE) {
```

Reply:
The thing is, from a user perspective, it doesn't make much sense to retrieve `rules` without the `defaults` and `declarations`. For example, this would error out:

https://github.com/rstudio/bslib/pull/160/files#diff-c3c3731e2b8667ea34c6a7c9625e6e755ae8ae02ff1527d418400f1052e71641R17

So, I think `TRUE` is a better default because it's "safer" in some sense